src/main/scala/mesosphere/marathon/core/task/tracker/InstanceTracker.scala

```diff
@@ -87,10 +87,11 @@ trait InstanceTracker extends StrictLogging {
   /**
     * An ongoing source of instance updates. On materialization, receives an update for all current instances
     */
-  val instanceUpdates: Source[(InstancesSnapshot, Source[InstanceChange, NotUsed]), NotUsed]
+  val instanceUpdates: InstanceTracker.InstanceUpdates
```

Comment:
Shouldn't this be implemented in `InstanceTrackerDelegate` or `InstanceTrackerAcotr`? I couldn't find the changes there.

Reply:
It's just a type alias